### ISSUE TITLE
Code for DAG experiment

### ISSUE THREAD

**Issue by rookiejackal** [agadetsky/pytorch-pl-variance-reduction#3]
Hello,

Thanks a lot for your code. A really interesting work. Will the code for experiments on DAGs be released? It would be very nice to see them on some real problems like causal inference as given in the paper. 

I tried to manually reproduce the results for 10, 20, 50 nodes but unfortunately failed to do so. If the code is not being released for it, can I kindly know some details regarding the implementation of accelerated proximal method (hyperparmeters, optimizer used, number of iterations run, learning rate etc.).

Thanks a lot for your help.

**Comment by agadetsky** [comment]
@rookiejackal Hi!

We don't plan to release the code for DAG experiments in the nearby future. But you can e-mail me with specific questions you have about the implementation.
